### ISSUE TITLE
Docker-compose ps not showing any output

### ISSUE THREAD

**Issue by crhuber** [docker/compose#2188]
I am trying to run docker-compose ps and docker-compose logs and neither are showing any output. 

    > sudo docker-compose -f /opt/docker-compose/server1-compose.yml ps
    Name   Command   State   Ports 
    ------------------------------

    > sudo docker-compose -f /opt/docker-compose/server1-compose.yml logs
    Attaching to 

Both commands are returning intended output. What is wrong here?
docker-compose version: 1.4.2
Docker version 1.7.1, build 786b29d

**Comment by feliperuhland** [comment]
@crhuber, Is there any container running? What is the output of ``docker ps``?

**Comment by crhuber** [comment]
yes i see all the containers running as expected from the compose file. So i verified they are running but compose ps shows no output

**Comment by dnephin** [comment]
I suspect what's happening here is that the project name is different. The default project name is the basename of a directory, so if you run `docker-compose` from a different directory you might get a different project name.

You can set it with either `-p` or `COMPOSE_PROJECT_NAME` environment variable.  If you look at the first part of the container names (before the first underscore) that's the project name.

There are open issues to configure the project name from a file. I think we'll be looking to implement that soon.

**Comment by crhuber** [comment]
@dnephin you were exactly right sir. thank you for figuring it out. Adding the -p switch to my compose command fixed the issue. 
ie: sudo docker-compose -f /opt/docker-compose/server1-compose.yml logs -p projectname

**Comment by yuklia** [comment]
I faced the same issue as @crhuber docker-compose version 1.7.1, build 0a9ab35.
Steps to reproduce:
1 I am on folder with docker-compose.yml
2 Run docker-compose -p projname up -d
3 docker-compose ps
Name   Command   State   Ports 
------------------------------
**4 docker ps shows all running services**

**Comment by yicone** [comment]
@yuklia 

You can try this:
docker-compose -p projname ps

docker-compose version 1.7.1, build 0a9ab35

**Comment by natekwilson** [comment]
I ran into this same problem! Except I had the COMPOSE_PROJECT_NAME env var set in my .env file with interspersed **Capitals** while docker-compose used an auto-formatted version of the string to using all lowercase. 

Under certain conditions running the CLI tool the .env variable caused a mis-match and it showed empty results no matter what. 

Once I changed the COMPOSE_PROJECT_NAME var in my .env to all lowercase it worked again.
